lib/mix/lib/mix/tasks/help.ex

```diff
@@ -4,11 +4,12 @@ defmodule Mix.Tasks.Help do
   @shortdoc "Prints help information for tasks"
 
   @moduledoc """
-  Lists all tasks or prints the documentation for a given task.
+  Lists all aliases and tasks or prints the documentation for a given task.
```

Comment:
Minor nitpick: I would change the order of "tasks" and "aliases" here (and below). For some reason, in my head, it makes sense to put "tasks" at the beginning as they are more relevant for this specific task :) So, "Lists all tasks and aliases or prints the documentation for a given task or alias"